pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -97,48 +116,51 @@ type CiliumBGPNeighbor struct {
 	// +kubebuilder:validation:Minimum=1
 	// +kubebuilder:validation:Maximum=65535
 	// +kubebuilder:default=179
-	PeerPort *int `json:"peerPort,omitempty"`
+	PeerPort *int32 `json:"peerPort,omitempty"`
 	// PeerASN is the ASN of the peer BGP router.
 	// Supports extended 32bit ASNs
 	//
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4294967295
-	PeerASN int `json:"peerASN"`
+	PeerASN int64 `json:"peerASN"`
 	// EBGPMultihopTTL controls the multi-hop feature for eBGP peers.
 	// Its value defines the Time To Live (TTL) value used in BGP packets sent to the neighbor.
-	// When empty or zero, eBGP multi-hop feature is disabled. The value is ignored for iBGP peers.
+	// The value 1 implies that eBGP multi-hop feature is disabled. This field is ignored for iBGP peers.
```

Comment:
```suggestion
	// The value 1 implies that eBGP multi-hop feature is disabled (because only a single hop is allowed). This field is ignored for iBGP peers.
```

Small clarification request?